pinot-core/src/test/java/org/apache/pinot/core/common/datatable/DataTableSerDeTest.java

```diff
@@ -417,10 +424,26 @@ private void fillDataTableWithRandomData(DataTableBuilder dataTableBuilder,
             DOUBLES[rowId] = RANDOM.nextDouble();
             dataTableBuilder.setColumn(colId, DOUBLES[rowId]);
             break;
+          case BIG_DECIMAL:
+            BIG_DECIMALS[rowId] = BigDecimal.valueOf(RANDOM.nextInt());
```

Comment:
May be use an actual large unscaled value instead of INT ?

Reply:
BigDecimal has virtually no limit. However `RANDOM.nextDouble()`. 
the goal here is not to test the arithmetic of BigDecimal but rather the variable length ser/de, so I think double should cover a good variety. thoughts?

Reply:
I see. My point was actually around to use BIG_DECIMAL of variable / arbitrary length for DataTableSerDe test as opposed to fix width int or double

Reply:
yes. according to the class def:
- BigDecimal = BigInteger + scale
- BigInteger = bitCount + bitLength + firstNonzeroIntNum + lowestSetBit + signum + int[] mag (4+4+4+4+4+int[])

i think the mag should be dynamic enough (e.g. for using double as random input)